### ISSUE TITLE
May 2016 Enchanted Armoire equipment never delivered

### ISSUE THREAD

**Issue by TheSchwamm** [HabitRPG/habitica#7146]
General Info
- UUID: c7966a3e-1d16-4cd9-8bea-6ed10679f6ba
- Browser: Firefox/Chrome
- OS: Windows of various flavors
### Description

Hello!

While i did see Bailey's announcement that the May Enchanted Armoire gear is available, the items have never appeared.  Hovering over my Enchanted Armoire tells me that new equipment will be available during the first week of the month.

i've tried reloading the browser, restarting the browser, using a different browser on a different computer, and nothing.  Posting to the Tavern seems to show a scattering of other folks with the same problem, so here i am with a bug report.
#### Console Errors

N/A

**Comment by Alys** [comment]
Sorry about that! I've reset the flag in your account that tells you when the Armoire has more gear.

Reload to see the correct Armoire message, but tell me if it still isn't correct!

For our reference, I saw (and fixed) another report of this in the Tavern, so I'm leaving this open so that we can improve the process after API v3 is released.

**Comment by TheSchwamm** [comment]
That did seem to work!  i'm just confused about how some people clearly had that flag set while others didn't.

**Comment by Alys** [comment]
@TheSchwamm There's a script that is run over all user accounts to set the flag, but there must have been timeouts while it was running, so that some accounts were missed.

**Comment by TheSchwamm** [comment]
Welp, i guess we've found a script that needs some error checking.  Happy
to help, haha!

**Comment by Alys** [comment]
Out of all the users who logged in since the beginning of April, I've found about 3760 who had `flags.armoireEmpty` set to true when it should be false (i.e., those people were being told that the Armoire was empty when they did not yet own all of the most recent Armoire items). I've flipped the flag for them so they should find that the Armoire tells them the correct thing now.

There's likely other users as well - I searched for only those who logged in recently and didn't have all off this month's gear. I'll do a more thorough search later this weekend if I have time.
